tests/components/frontend/test_init.py

```diff
@@ -484,3 +484,12 @@ async def test_get_version(hass, hass_ws_client):
     assert msg["type"] == TYPE_RESULT
     assert msg["success"]
     assert msg["result"] == {"version": cur_version}
+
+
+async def test_static_paths(hass, mock_http_client):
+    """Test static paths."""
+    resp = await mock_http_client.get(
+        "/.well-known/change-password", allow_redirects=False
+    )
+    assert resp.status == 302
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6103615)